pkg/inference/backends/llamacpp/llamacpp_config.go

```diff
@@ -57,25 +53,27 @@ func (c *Config) GetArgs(model types.Model, socket string, mode inference.Backen
 		args = append(args, "--embeddings")
 	}
 
-	args = append(args, "--ctx-size", strconv.FormatUint(GetContextSize(&modelCfg, config), 10))
+	// Add context size from model config or backend config
+	args = append(args, "--ctx-size", strconv.FormatUint(GetContextSize(bundle.RuntimeConfig(), config), 10))
 
 	// Add arguments from backend config
 	if config != nil {
 		args = append(args, config.RuntimeFlags...)
 	}
 
 	// Add arguments for Multimodal projector
-	path, err := model.MMPROJPath()
-	if path != "" && err == nil {
+	mmprojPath := bundle.MMPROJPath()
+	fmt.Println("GOT MMPROJ:", mmprojPath)
```

Comment:
nit: looks like a leftover debug print. Could be useful as an `c.log.Info()`, though the runtime args should be logged somewhere around the point of execution.

Reply:
oops! thank you